.github/scripts/label_promoted_commits.py

```diff
@@ -30,12 +30,29 @@ def parser():
 
 def add_comment_and_close_pr(pr_number, repository, comment):
     pr = repository.get_pull(pr_number)
-    if pr.state == open:
+    if 'open' in pr.state:
```

Comment:
thanks for fixing this!

but can we be less tolerant here ? as per the response schema, 

```yaml
"state": {
      "description": "State of this Pull Request. Either `open` or `closed`.",
      "enum": [
        "open",
        "closed"
      ],
      "type": "string",
      "examples": [
        "open"
      ]
    },
```
`state` can only be `"open"` or `"closed"`.  see https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#get-a-pull-request